src/app/components/MediaLoader/utils/buildSettings.ts

```diff
@@ -38,6 +43,7 @@ const buildSettings = ({
     enableToucan: true,
     appType: isAmp ? 'amp' : 'responsive',
     appName: service !== 'news' ? `news-${service}` : 'news',
+    externalEmbedUrl: '',
```

Comment:
This should actually be removed, as On Demand TV pages cannot be embedded. The reason the test was failing was due to a merge conflict being incorrectly resolved (my bad oops), so let's remove this from here, and remove `externalEmbedUrl: ''` from the test as well.

Reply:
Ahh, I see. Thanks for letting me know.